packages/plugins/ReverseGeocoder/README.md

```diff
@@ -20,6 +20,19 @@ This module has been written for the HH WPS service. The return format is custom
 | zoomTo | number? | If specified, plugin zooms to given coordinate after successful reverse geocoding; number indicates maximal zoom level. |
 
 
+Example configuration:
+```js
+
+reverseGeocoder: {
```

Comment:
Please add an example for `addressTarget`

Reply:
https://github.com/Dataport/polar/pull/158/commits/666d8d9345c177c6d0876ad603a24671780f8b8c